docs/serverless-v2/01-02-serverless-v2.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Architecture
+---
+
+Serverless v2 replies on [Knative Serving](https://knative.dev/docs/serving/) for deploying and managing functions, and [Tekton](https://github.com/tektoncd/pipeline) as a pipeline for creating and running on-cluster processes. See how these and other resources process a lambda within a Kyma cluster:
+
+![Serverless v2 architecture](./assets/serverless-v2-architecture.svg)
+
+1. The user creates a lambda either through the UI or by applying a Function custom resource (CR). This CR contains the lambda definition (business logic that the user wants to execute) and information on the environment on which it should be run (Node 6 or Node 8).
+
+2. Function Controller (FC) detects a new Function CR and reads its definition.
+
+3. Based on the Function CR definition, FC creates a [TaskRun CR](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md), the purpose of which is to create an image based on the defined lambda.
+
+4. Tekton Controller (TC) detects the new TaskRun CR and reads its definition.
+
+5. Based on the TaskRun CR definition, TC triggers a pipeline that creates an image with lambda definition (Dockerfile). It later pushes this image to the local registry.
```

Comment:
`local registry` - I think that we should present it that way: in default user pushes to some docker.hub or gcr.io, and we have a fallback to internal registry (WIP)

Reply:
Just say to the registry, without details like local, remote etc....

Reply:
👍